internal/providers/terraform/azure/mariadb_server.go

```diff
@@ -0,0 +1,122 @@
+package azure
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetAzureMariaDBServerRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_mariadb_server",
+		RFunc: NewAzureMariaDBServer,
+	}
+}
+
+func NewAzureMariaDBServer(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var costComponents []*schema.CostComponent
+
+	region := d.Get("location").String()
+	sku := d.Get("sku_name").String()
+	var tier, family, cores string
+	if s := strings.Split(sku, "_"); len(s) == 3 {
+		tier = strings.Split(sku, "_")[0]
+		family = strings.Split(sku, "_")[1]
+		cores = strings.Split(sku, "_")[2]
+	} else {
+		log.Println("MariaDB sku format is not recognised. Please follows the tier_family_cores pattern (for example: GP_Gen5_4)")
```

Comment:
log.Warnf("
```suggestion
		log.Warnf("Unrecognised MariaDB SKU format for resource %s: %s", d.Address, sku)
```

I think we should output the address and sku.